app/src/main/java/com/locoquest/app/dto/Benchmark.kt

```diff
@@ -1,5 +1,8 @@
 package com.locoquest.app.dto
 
+import androidx.room.Entity
+
+@Entity(tableName="benchmark")
```

Comment:
Added @entity to call @query from ILocalBenchmmarkDAO
https://developer.android.com/reference/kotlin/androidx/room/Entity

Reply:
Is this currently used in Room?